marquez_airflow/dag.py

```diff
@@ -40,8 +42,16 @@ def create_dagrun(self, *args, **kwargs):
         try:
             marquez_jobrun_id = self.report_jobrun(run_args,
                                                    kwargs['execution_date'])
+            log.info(f'Successfully recorded job run.',
+                     airflow_dag_id=self.dag_id,
+                     marquez_run_id=marquez_jobrun_id,
+                     marquez_namespace=self.marquez_namespace)
         except Exception as e:
-            logging.warning("[Marquez]\t{}".format(e))
+            log.error(f'Failed to record job run: {e}',
+                      airflow_dag_id=self.dag_id,
+                      marquez_namespace=self.marquez_namespace)
+            pass
```

Comment:
nit: Necessary?

Reply:
I added `pass` more for documentation purposes as we aren't (re)raising an error.